File: agent_misc/mcps/codeql-development-mcp-server/server/ql/javascript/tools/test/CallGraphTo/Example1.js
// COMPLIANT: Functions that should not be analyzed
function unrelated1() {
    // No calls
}

function caller1() {
    targetFunc();
}

function caller2() {
    targetFunc();
}

// NON_COMPLIANT: Target function for call graph analysis
function targetFunc() {
    unrelated1();
}

File: agent_misc/mcps/codeql-development-mcp-server/server/ql/swift/tools/test/CallGraphTo/Example1.swift
// COMPLIANT: Functions that should not be analyzed
func unrelated1() {
    // No calls
}

// NON_COMPLIANT: Target function for call graph analysis
func targetFunc() {
    unrelated1()
}

func caller1() {
    targetFunc()
}

func caller2() {
    targetFunc()
}
